### PR TITLE
getSessionIdByEventId

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: Gateway/SessionIdsProxy.fs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿module SessionIdsProxy
+
+open Config
+open Dtos
+open JsonHttpClient
+open RestModels
```
Comment (niksoper) [inline Gateway/SessionIdsProxy.fs]: It doesn't look like we need Dtos or RestModels here.

File: Gateway/SessionIdsProxy.fs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿module SessionIdsProxy
+
+open Config
+open JsonHttpClient
+open System
+
+let getSessionIdsByEventId (eventId : Guid) = get<Guid []> <| new Uri(sessionIdsUri, "?eventId=" + eventId.ToString())
```
Comment (pallenSL) [inline Gateway/SessionIdsProxy.fs]: Newline, please.

File: Gateway/app.config [hunk 3]
```diff
@@ -19,6 +19,7 @@
     <add key="LastContactUrl" value="http://comms:8080/last-contact/"/>
     <add key="EventsUrl" value="http://sessions:8080/events/"/>
     <add key="SessionsUrl" value="http://sessions:8080/sessions/"/>
+    <add key="SessionIdsUrl" value="http://sessions:8080/sessionId/"/>
```
Comment (pallenSL) [inline Gateway/app.config]: Why isn't this a plural?

File: Gateway/app.config [hunk 4]
```diff
@@ -27,6 +28,7 @@
     <add key="LastContactUrl" value="http://localhost:8082/last-contact/"/>
     <add key="EventsUrl" value="http://localhost:8083/events/"/>
     <add key="SessionsUrl" value="http://localhost:8083/sessions/"/>
+    <add key="SessionIdsUrl" value="http://localhost:8083/sessionId/"/>
```
Comment (pallenSL) [inline Gateway/app.config]: Notaplural.

Comment (jebbinSCL) [inline Gateway/app.config]: Ah, The url isn’t plural on the sessions service either. I’ll do that as a separate pull request, since sessions needs to be changed as well.

File: Gateway/SessionIdsProxy.fs (hunk 2 above)
Comment (jebbinSCL) [inline Gateway/SessionIdsProxy.fs]: That would be different formatting from the rest of the app for short functions, and what is normal for F# ?

File: Gateway/app.config (hunk 3 above)
Comment (jebbinSCL) [inline Gateway/app.config]: Ah, The url isn’t plural on the sessions service either. I’ll do that as a separate pull request, since sessions needs to be changed as well.

Comment (jebbinSCL) [inline Gateway/app.config]: I've added a jira item for it.

File: Gateway/SessionIdsProxy.fs (hunk 2 above)
Comment (niksoper) [inline Gateway/SessionIdsProxy.fs]: I think this is just about within the limits of acceptable line length (119 chars).  More than 120 means trouble.

**Comment by niksoper** [comment]
:+1:

File: Gateway/app.config (hunk 3 above)
Comment (jebbinSCL) [inline Gateway/app.config]: Just realised the session code isn't merged yet, so i'll make that change now.

**Comment by jebbinSCL** [comment]
Adjusted urls

File: Gateway/SessionIdsProxy.fs (hunk 2 above)
Comment (pallenSL) [inline Gateway/SessionIdsProxy.fs]: I meant at the end.
You know, that warning you must be seeing and ignoring on every single commit!

Comment (jebbinSCL) [inline Gateway/SessionIdsProxy.fs]: It's so tiny I don't see it unless I'm looking for it, and local git doesn't say anything about it.....

Comment (jebbinSCL) [inline Gateway/SessionIdsProxy.fs]: Adjusted.

**Comment by pallenSL** [comment]
Ok as a PR.  Unconvinced on sessionIds being a proper noun.
:shipit:
